### PR TITLE
Runs with single offset irf

### PR DESCRIPTION
**Pull request by sona-patel**

### CODE REVIEW

File: pyV2DL3/eventdisplay/convertROOT2fits.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ def convertROOT2fits(files, eff, **kwargs):
     force_extrapolation = kwargs.get("force_extrapolation", False)
     fuzzy_boundary = kwargs.get("fuzzy_boundary", 0.0)
 
-    if not(full_enclosure) and not(point_like):
+    if not (full_enclosure) and not (point_like):
```
[the remark this answers is not in the document]
Comment (sona-patel) [reply pyV2DL3/eventdisplay/convertROOT2fits.py:28]: True, we have only two! 

Note: The changes in `conertROOT2fits.py` and `v2dl3_for_vegas.py` is not related to this PR. flake8 formatting was required to pass the CI test.

[the remark this answers is not in the document]
Comment (GernotMaier) [reply pyV2DL3/eventdisplay/convertROOT2fits.py:28]: OK, so this if statement does not make sense? Is it even possible to reach this? If not, maybe we should remove it then.

[the remark this answers is not in the document]
Comment (sona-patel) [reply pyV2DL3/eventdisplay/convertROOT2fits.py:28]: I think this complete file should be removed as its not used at all in the converter - #149 !

[the remark this answers is not in the document]
Comment (GernotMaier) [reply pyV2DL3/eventdisplay/convertROOT2fits.py:28]: Yes, please remove it - I linked #149 to this PR.

**Review by GernotMaier — APPROVED** [review]
All good now from my side.

**Review by tobiaskleiner — APPROVED** [review]

**Review by GernotMaier — CHANGES_REQUESTED** [review]
@sona-patel : could you clarify the relation of this branch with the `rhv_runs` branch?

**Comment by GernotMaier** [comment]
This PR has been done to prepare a demonstration file for gammapy. It is superseeded and therefore closed.

**Pull request closed without merging**